module,execution_time (sec),success,timestamp,error,error_short
cowidev.vax.batch.argentina,3.81,True,2022-07-26T07:00:13,,
cowidev.vax.batch.australia,1.99,True,2022-07-26T07:00:11,,
cowidev.vax.batch.austria,,,2022-07-26T07:00:09,,
cowidev.vax.batch.belgium,2.7,True,2022-07-26T07:00:12,,
cowidev.vax.batch.bolivia,2.36,True,2022-07-26T07:00:12,,
cowidev.vax.batch.canada,7.08,False,2022-07-26T07:02:43,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 254, in main
    Canada().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 241, in export
    df_man = df_man.pipe(self.pipeline_manufacturer)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 172, in pipeline_manufacturer
    validate_vaccines(df, self.vaccine_mapping)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/utils/checks.py"", line 257, in validate_vaccines
    raise ValueError(f""Missing vaccines: {vaccines_wrong}"")
ValueError: Missing vaccines: {'Medicago Covifenz'}
",Missing vaccines: {'Medicago Covifenz'}
cowidev.vax.batch.chile,0.66,True,2022-07-26T07:00:12,,
cowidev.vax.batch.czechia,3.73,True,2022-07-26T07:00:16,,
cowidev.vax.batch.denmark,3.58,True,2022-07-26T07:00:16,,
cowidev.vax.batch.ecdc,60.09,False,2022-07-26T07:03:43,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connection.py"", line 174, in _new_conn
    conn = connection.create_connection(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/util/connection.py"", line 96, in create_connection
    raise err
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/util/connection.py"", line 86, in create_connection
    sock.connect(sa)
socket.timeout: timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 699, in urlopen
    httplib_response = self._make_request(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 382, in _make_request
    self._validate_conn(conn)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 1010, in _validate_conn
    conn.connect()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connection.py"", line 358, in connect
    conn = self._new_conn()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connection.py"", line 179, in _new_conn
    raise ConnectTimeoutError(
urllib3.exceptions.ConnectTimeoutError: (<urllib3.connection.HTTPSConnection object at 0x7fafebe8c040>, 'Connection to opendata.ecdc.europa.eu timed out. (connect timeout=30)')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/adapters.py"", line 440, in send
    resp = conn.urlopen(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 755, in urlopen
    retries = retries.increment(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/util/retry.py"", line 574, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='opendata.ecdc.europa.eu', port=443): Max retries exceeded with url: /covid19/vaccine_tracker/csv/data.csv (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fafebe8c040>, 'Connection to opendata.ecdc.europa.eu timed out. (connect timeout=30)'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/ecdc.py"", line 506, in main
    ECDC().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/ecdc.py"", line 496, in export
    df = self.read().pipe(self.pipe_base)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/ecdc.py"", line 111, in read
    df = read_csv_from_url(self.source_url, timeout=30)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/download.py"", line 39, in read_csv_from_url
    download_file_from_url(url, tmp.name, timeout=timeout, verify=verify, ciphers_low=ciphers_low)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/download.py"", line 52, in download_file_from_url
    r = requests.get(url, stream=True, timeout=timeout, verify=verify)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/api.py"", line 75, in get
    return request('get', url, params=params, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/api.py"", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/sessions.py"", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/sessions.py"", line 645, in send
    r = adapter.send(request, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/adapters.py"", line 507, in send
    raise ConnectTimeout(e, request=request)
requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='opendata.ecdc.europa.eu', port=443): Max retries exceeded with url: /covid19/vaccine_tracker/csv/data.csv (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fafebe8c040>, 'Connection to opendata.ecdc.europa.eu timed out. (connect timeout=30)'))
","HTTPSConnectionPool(host='opendata.ecdc.europa.eu', port=443): Max retries exceeded with url: /covid19/vaccine_tracker/csv/data.csv (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fafebe8c040>, 'Connection to opendata.ecdc.europa.eu timed out. (connect timeout=30)'))"
cowidev.vax.batch.ecuador,1.9,True,2022-07-26T07:00:15,,
cowidev.vax.batch.estonia,0.69,True,2022-07-26T07:00:16,,
cowidev.vax.batch.france,0.46,True,2022-07-26T07:00:16,,
cowidev.vax.batch.germany,1.92,True,2022-07-26T07:00:18,,
cowidev.vax.batch.greece,2.63,True,2022-07-26T07:00:19,,
cowidev.vax.batch.hong_kong,11.89,True,2022-07-26T07:00:28,,
cowidev.vax.batch.indonesia,,,2022-07-26T07:00:18,,
cowidev.vax.batch.ireland,1.0,True,2022-07-26T07:00:19,,
cowidev.vax.batch.israel,9.45,True,2022-07-26T07:00:27,,
cowidev.vax.batch.italy,9.54,True,2022-07-26T07:00:28,,
cowidev.vax.batch.japan,1.28,False,2022-07-26T07:03:45,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 241, in main
    Japan().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 225, in export
    df = self.read().pipe(self.pipeline_base)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 161, in read
    df_latest = self.read_latest().pipe(self.pipe_latest)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 91, in read_latest
    dfs.append(self._read_xlsx(self.source_url, self.sheets, self.metrics))
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 102, in _read_xlsx
    xlsx = pd.ExcelFile(tmp.name)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/io/excel/_base.py"", line 1195, in __init__
    raise ValueError(
ValueError: Excel file format cannot be determined, you must specify an engine manually.
","Excel file format cannot be determined, you must specify an engine manually."
cowidev.vax.batch.jersey,1.06,True,2022-07-26T07:00:28,,
cowidev.vax.batch.latvia,18.35,True,2022-07-26T07:00:46,,
cowidev.vax.batch.lithuania,1.56,True,2022-07-26T07:00:30,,
cowidev.vax.batch.luxembourg,1.48,True,2022-07-26T07:00:30,,
cowidev.vax.batch.malaysia,0.46,True,2022-07-26T07:00:29,,
cowidev.vax.batch.malta,0.95,True,2022-07-26T07:00:30,,
cowidev.vax.batch.netherlands,,,2022-07-26T07:00:30,,
cowidev.vax.batch.new_zealand,20.71,True,2022-07-26T07:00:50,,
cowidev.vax.batch.norway,0.31,True,2022-07-26T07:00:30,,
cowidev.vax.batch.peru,3.39,True,2022-07-26T07:00:33,,
cowidev.vax.batch.portugal,,,2022-07-26T07:00:30,,
cowidev.vax.batch.romania,6.81,True,2022-07-26T07:00:37,,
cowidev.vax.batch.saudi_arabia,4.03,True,2022-07-26T07:00:37,,
cowidev.vax.batch.singapore,1.67,True,2022-07-26T07:00:38,,
cowidev.vax.batch.slovakia,45.22,True,2022-07-26T07:01:22,,
cowidev.vax.batch.slovenia,2.2,True,2022-07-26T07:00:41,,
cowidev.vax.batch.south_korea,42.6,True,2022-07-26T07:01:23,,
cowidev.vax.batch.spc,4.35,False,2022-07-26T07:03:49,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/spc.py"", line 220, in main
    SPC().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/spc.py"", line 214, in export
    data = self.read()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/spc.py"", line 59, in read
    return self.parse_data(data)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/spc.py"", line 73, in parse_data
    return self._build_df_list(vaccination_data)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/spc.py"", line 103, in _build_df_list
    data[k] = self._build_df(v, k)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/spc.py"", line 135, in _build_df
    df = df.pipe(make_monotonic)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/utils/utils.py"", line 44, in make_monotonic
    raise Exception(
Exception: 50 rows have been removed. That is more than maximum allowed (10) by make_monotonic() - check the data. Check 
          date  ...  vaccine
9   2021-07-13  ...      NaN
10  2021-07-20  ...      NaN
11  2021-07-27  ...      NaN
12  2021-08-02  ...      NaN
13  2021-08-09  ...      NaN
14  2021-08-16  ...      NaN
15  2021-08-23  ...      NaN
16  2021-08-31  ...      NaN
17  2021-09-07  ...      NaN
18  2021-09-14  ...      NaN
19  2021-09-21  ...      NaN
20  2021-09-28  ...      NaN
21  2021-10-05  ...      NaN
22  2021-10-12  ...      NaN
23  2021-10-18  ...      NaN
24  2021-10-25  ...      NaN
25  2021-11-01  ...      NaN
26  2021-11-08  ...      NaN
27  2021-11-15  ...      NaN
28  2021-11-22  ...      NaN
29  2021-11-30  ...      NaN
30  2021-12-06  ...      NaN
31  2021-12-13  ...      NaN
32  2021-12-20  ...      NaN
33  2022-01-04  ...      NaN
34  2022-01-11  ...      NaN
35  2022-01-18  ...      NaN
36  2022-01-24  ...      NaN
37  2022-01-31  ...      NaN
38  2022-02-07  ...      NaN
39  2022-02-14  ...      NaN
40  2022-02-21  ...      NaN
41  2022-02-28  ...      NaN
42  2022-03-07  ...      NaN
43  2022-03-14  ...      NaN
44  2022-03-21  ...      NaN
45  2022-03-28  ...      NaN
46  2022-04-04  ...      NaN
47  2022-04-11  ...      NaN
48  2022-04-18  ...      NaN
49  2022-04-25  ...      NaN
50  2022-05-02  ...      NaN
51  2022-05-09  ...      NaN
52  2022-05-16  ...      NaN
53  2022-05-23  ...      NaN
54  2022-05-30  ...      NaN
55  2022-06-06  ...      NaN
56  2022-06-13  ...      NaN
57  2022-06-20  ...      NaN
58  2022-06-27  ...      NaN

[50 rows x 8 columns]
","50 rows have been removed. That is more than maximum allowed (10) by make_monotonic() - check the data. Check 
          date  ...  vaccine
9   2021-07-13  ...      NaN
10  2021-07-20  ...      NaN
11  2021-07-27  ...      NaN
12  2021-08-02  ...      NaN
13  2021-08-09  ...      NaN
14  2021-08-16  ...      NaN
15  2021-08-23  ...      NaN
16  2021-08-31  ...      NaN
17  2021-09-07  ...      NaN
18  2021-09-14  ...      NaN
19  2021-09-21  ...      NaN
20  2021-09-28  ...      NaN
21  2021-10-05  ...      NaN
22  2021-10-12  ...      NaN
23  2021-10-18  ...      NaN
24  2021-10-25  ...      NaN
25  2021-11-01  ...      NaN
26  2021-11-08  ...      NaN
27  2021-11-15  ...      NaN
28  2021-11-22  ...      NaN
29  2021-11-30  ...      NaN
30  2021-12-06  ...      NaN
31  2021-12-13  ...      NaN
32  2021-12-20  ...      NaN
33  2022-01-04  ...      NaN
34  2022-01-11  ...      NaN
35  2022-01-18  ...      NaN
36  2022-01-24  ...      NaN
37  2022-01-31  ...      NaN
38  2022-02-07  ...      NaN
39  2022-02-14  ...      NaN
40  2022-02-21  ...      NaN
41  2022-02-28  ...      NaN
42  2022-03-07  ...      NaN
43  2022-03-14  ...      NaN
44  2022-03-21  ...      NaN
45  2022-03-28  ...      NaN
46  2022-04-04  ...      NaN
47  2022-04-11  ...      NaN
48  2022-04-18  ...      NaN
49  2022-04-25  ...      NaN
50  2022-05-02  ...      NaN
51  2022-05-09  ...      NaN
52  2022-05-16  ...      NaN
53  2022-05-23  ...      NaN
54  2022-05-30  ...      NaN
55  2022-06-06  ...      NaN
56  2022-06-13  ...      NaN
57  2022-06-20  ...      NaN
58  2022-06-27  ...      NaN

[50 rows x 8 columns]"
cowidev.vax.batch.sweden,12.69,True,2022-07-26T07:01:03,,
cowidev.vax.batch.switzerland,31.67,True,2022-07-26T07:01:24,,
cowidev.vax.batch.trinidad_and_tobago,1.57,True,2022-07-26T07:01:05,,
cowidev.vax.batch.ukraine,,,2022-07-26T07:01:05,,
cowidev.vax.batch.united_kingdom,9.52,True,2022-07-26T07:01:14,,
cowidev.vax.batch.united_states,17.31,True,2022-07-26T07:01:30,,
cowidev.vax.batch.uruguay,3.44,True,2022-07-26T07:01:18,,
cowidev.vax.batch.zimbabwe,0.61,True,2022-07-26T07:01:18,,
cowidev.vax.incremental.africacdc,,,2022-07-26T07:01:18,,
cowidev.vax.incremental.antigua_barbuda,4.01,True,2022-07-26T07:01:22,,
cowidev.vax.incremental.aruba,1.17,True,2022-07-26T07:01:23,,
cowidev.vax.incremental.azerbaijan,7.1,True,2022-07-26T07:01:29,,
cowidev.vax.incremental.bahrain,1.37,True,2022-07-26T07:01:25,,
cowidev.vax.incremental.bangladesh,15.0,True,2022-07-26T07:01:38,,
cowidev.vax.incremental.barbados,2.4,True,2022-07-26T07:01:26,,
cowidev.vax.incremental.brazil,0.15,True,2022-07-26T07:01:25,,
cowidev.vax.incremental.bulgaria,1.03,True,2022-07-26T07:01:26,,
cowidev.vax.incremental.china,50.33,True,2022-07-26T07:04:39,,
cowidev.vax.incremental.costa_rica,,,2022-07-26T07:01:26,,
cowidev.vax.incremental.croatia,0.14,True,2022-07-26T07:01:26,,
cowidev.vax.incremental.cuba,3.17,False,2022-07-26T07:04:43,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/cuba.py"", line 73, in main
    Cuba().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/cuba.py"", line 59, in export
    data = self.read().pipe(self.pipeline)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/cuba.py"", line 25, in read
    return self._parse_data(soup)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/cuba.py"", line 30, in _parse_data
    ""date"": self._parse_date(soup),
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/cuba.py"", line 37, in _parse_date
    date_str = match.group(1)
AttributeError: 'NoneType' object has no attribute 'group'
",'NoneType' object has no attribute 'group'
cowidev.vax.incremental.curacao,0.42,True,2022-07-26T07:01:30,,
cowidev.vax.incremental.cyprus,0.71,True,2022-07-26T07:01:31,,
cowidev.vax.incremental.dominican_republic,9.99,True,2022-07-26T07:01:40,,
cowidev.vax.incremental.el_salvador,1.32,True,2022-07-26T07:01:32,,
cowidev.vax.incremental.faeroe_islands,0.33,True,2022-07-26T07:01:32,,
cowidev.vax.incremental.fiji,0.0,True,2022-07-26T07:01:32,,
cowidev.vax.incremental.finland,0.2,True,2022-07-26T07:01:32,,
cowidev.vax.incremental.gabon,,,2022-07-26T07:01:32,,
cowidev.vax.incremental.georgia,0.54,True,2022-07-26T07:01:32,,
cowidev.vax.incremental.greenland,0.58,True,2022-07-26T07:01:33,,
cowidev.vax.incremental.guatemala,,,2022-07-26T07:01:32,,
cowidev.vax.incremental.guernsey,0.16,True,2022-07-26T07:01:33,,
cowidev.vax.incremental.hungary,0.86,True,2022-07-26T07:01:33,,
cowidev.vax.incremental.iceland,0.21,True,2022-07-26T07:01:33,,
cowidev.vax.incremental.india,1.0,True,2022-07-26T07:01:34,,
cowidev.vax.incremental.iran,2.48,True,2022-07-26T07:01:36,,
cowidev.vax.incremental.isle_of_man,0.92,True,2022-07-26T07:01:35,,
cowidev.vax.incremental.jamaica,2.28,True,2022-07-26T07:01:37,,
cowidev.vax.incremental.kazakhstan,8.93,True,2022-07-26T07:01:45,,
cowidev.vax.incremental.kosovo,1.26,True,2022-07-26T07:01:38,,
cowidev.vax.incremental.kyrgyzstan,0.83,True,2022-07-26T07:01:39,,
cowidev.vax.incremental.laos,2.08,True,2022-07-26T07:01:40,,
cowidev.vax.incremental.macao,16.35,True,2022-07-26T07:01:55,,
cowidev.vax.incremental.moldova,1.18,True,2022-07-26T07:01:41,,
cowidev.vax.incremental.monaco,26.55,True,2022-07-26T07:02:07,,
cowidev.vax.incremental.mongolia,1.23,True,2022-07-26T07:01:43,,
cowidev.vax.incremental.montenegro,0.1,True,2022-07-26T07:01:43,,
cowidev.vax.incremental.morocco,,,2022-07-26T07:01:43,,
cowidev.vax.incremental.myanmar,,,2022-07-26T07:01:43,,
cowidev.vax.incremental.nepal,,,2022-07-26T07:01:43,,
cowidev.vax.incremental.north_macedonia,,,2022-07-26T07:01:43,,
cowidev.vax.incremental.northern_cyprus,0.82,True,2022-07-26T07:01:44,,
cowidev.vax.incremental.paho,50.9,True,2022-07-26T07:02:35,,
cowidev.vax.incremental.pakistan,1.61,True,2022-07-26T07:01:46,,
cowidev.vax.incremental.philippines,,,2022-07-26T07:01:46,,
cowidev.vax.incremental.poland,0.26,True,2022-07-26T07:01:47,,
cowidev.vax.incremental.qatar,,,2022-07-26T07:01:47,,
cowidev.vax.incremental.russia,0.85,False,2022-07-26T07:04:43,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/russia.py"", line 64, in main
    data = read(source).pipe(pipeline)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/russia.py"", line 11, in read
    soup = get_soup(source)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 83, in get_soup
    response = get_response(source, request_method, use_proxy, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 52, in get_response
    raise ValueError(
ValueError: Source https://gogov.ru/articles/covid-v-stats not reached! Error code 502 Bad Gateway
",Source https://gogov.ru/articles/covid-v-stats not reached! Error code 502 Bad Gateway
cowidev.vax.incremental.saint_lucia,,,2022-07-26T07:01:48,,
cowidev.vax.incremental.serbia,6.25,True,2022-07-26T07:01:54,,
cowidev.vax.incremental.south_africa,5.23,True,2022-07-26T07:01:59,,
cowidev.vax.incremental.spain,3.81,True,2022-07-26T07:01:59,,
cowidev.vax.incremental.sri_lanka,,,2022-07-26T07:01:59,,
cowidev.vax.incremental.suriname,15.11,True,2022-07-26T07:02:14,,
cowidev.vax.incremental.taiwan,7.49,True,2022-07-26T07:02:07,,
cowidev.vax.incremental.thailand,1.95,True,2022-07-26T07:02:09,,
cowidev.vax.incremental.turkey,0.46,True,2022-07-26T07:02:07,,
cowidev.vax.incremental.united_arab_emirates,,,2022-07-26T07:02:07,,
cowidev.vax.incremental.vietnam,,,2022-07-26T07:02:07,,
cowidev.vax.incremental.who,3.9,True,2022-07-26T07:02:11,,
cowidev.vax.incremental.zambia,0.8,True,2022-07-26T07:02:10,,
